src/domains/charts/chartMenuGroup/title.js

```diff
@@ -7,8 +7,7 @@ export const ChartMenuGroupIcon = ({ icon, ...rest }) => (
 )
 
 export const ChartMenuGroupTitle = ({ icon, title, ...rest }) => (
-  <Flex as={H0} color="key" gap={2} {...rest}>
-    <ChartMenuGroupIcon icon={icon} />
+  <Flex as={H0} color="text" gap={2} {...rest}>
```

Comment:
Let's remove the span below as well. Is it needed?

Reply:
Not sure yet!
After syncing with design, I will proceed with removing it (and maybe use only an `H0` instead of `Flex` or I will revert the removal of icon...! 😄

Reply:
@novykh after syncing with design, we are indeed removing the icon, only from `cloud-frontend` headings, so I have proceeded with the replacement of components